code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -49,6 +49,7 @@
 	key = "scream"
 	key_third_person = "screams"
 	message = "screams!"
+	message_mime = "acts out a scream!"
```

Comment:
From the wiki 
![image](https://user-images.githubusercontent.com/58055496/106060096-7d6d0100-60a8-11eb-8c3b-b0c4065ab246.png)
Could you give this more flavor? If we want players to mime to a high bar we should set it for them, right?

Reply:
This is a direct lift from the original living/emote.dm, I am simply restoring lost function.

![image](https://user-images.githubusercontent.com/3204033/106060696-f9237980-60b9-11eb-9220-019da3d857ef.png)


I am planning on adding more mime_emotes for audible emotes. I'd prefer to keep this atomized and a fix.

Reply:
If it were me adding the scream for the first time I would do "screams silently", so not exactly sure what you want from a generic hotkey emote.

Reply:
Our other `message_mimes`, for reference
*"giggles silently!"
*"appears to groan!"
*"laughs silently!"
*"sleeps soundly." (for snore)
*"appears hurt." (for whimper)

In my opinion miming (especially via hotkeys/baked in emotes) is more about using emote chains than it is about any one emote being quality. The issue here is that the scream should be silent, not that it should win a Hugo Nebula award.

Reply:
ah fuck it. Oh also yeah we should keep the var for the living emote, can't hurt anything

Reply:
The thing that really kills me about our emotes is that `adverbs` are kinda bad writing but we sort of need them because they are quick and effective? I'd really like to polish up a lot of our emotes.